src/components/MobileNavigation/MobileNavigation.scss

```diff
@@ -0,0 +1,42 @@
+.MobileNavigation__wrapper {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 32px;
+    gap: 50px;
+    height: 798px;
+    left: 0px;
+    top: 96px;
+    background: var(--color-white-lilac);
+    border-bottom: 1px solid #e3e0ed;
+
```

Comment:
I would say, you don't need box-sizing property here (it is already in global styles).

I don't think we need top/left properties here. But logo + hamburger menu bar will push the wrapper down anyway. 
also border-bottom- not sure what it is for.